apps/test/unit/templates/InlineDropdownMenuTest.js

```diff
@@ -0,0 +1,78 @@
+import {expect} from '../../util/reconfiguredChai';
+import React from 'react';
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import {InlineDropdownMenu} from '@cdo/apps/templates/InlineDropdownMenu';
+// import { expect } from 'chai';
+
+const clickSpy = sinon.spy();
+
+const defaultProps = {
+  icon: 'icon',
+  children: [<a href="foo">href</a>, <a onClick={clickSpy}>onclick</a>]
+};
+
+describe('InlineDropdownMenu', () => {
+  beforeEach(() => clickSpy.resetHistory());
+
+  it('is initially just a button', () => {
+    const wrapper = shallow(<InlineDropdownMenu {...defaultProps} />);
+    expect(wrapper.children().length).to.equal(1);
+    expect(wrapper.childAt(0).name()).to.equal('button');
+  });
+
+  it('is hidden if there are no children', () => {
+    const wrapper = shallow(<InlineDropdownMenu icon="icon" />);
+    expect(wrapper.children().length).to.equal(0);
+    expect(wrapper).to.deep.equal({});
+  });
+
+  it('shows children when clicked', () => {
```

Comment:
this test and the test below have the same name, is that intentional?

Reply:
oops. Thanks!